extensions/azurePublish-new/src/components/AzureProvisionDialog.tsx

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+
+import { ChooseProvisionAction } from './ChooseProvisionAction';
+
+export const AzureProvisionDialog = () => {
```

Comment:
Because this UI is hosted within a dialog in the client, consider naming it AzureProvisionWizard to avoid indicating it is controlling the modal frame.